.github/workflows/python-client-test-publish.yml

```
@@ -0,0 +1,109 @@
+      - dev
+    paths:
+      - 'clients/python-client/**'
+      - '.github/workflows/python-client-test-publish.yml'
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
+    
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)